vcpkg/users/platforms/windows.md

```diff
@@ -0,0 +1,166 @@
+
+There are also a number of [community triplets](https://github.com/microsoft/vcpkg/tree/master/triplets/community) for the MSVC compiler toolset as well.
+
+| Architecture | vcpkg community triplets |
+|--------------|--------------------------|
+| x64          | x64-windows-static-md    |
+| x86          | x86-windows-static       |
+|              | x86-windows-static-md    |
+| arm          | arm-windows              |
+|              | arm-windows-static       |
+| arm64        | arm64-windows-static     |
+|              | arm64-windows-static-md  |
+|              | arm64ec-windows          |
+
+> [!NOTE]
+> Community triplets are not tested as part of vcpkg repository's CI process, so regressions can occur as part of library updates. PRs improving support are welcome!
+
+The ``static`` linking triplets are set to use the MSVC Runtime as a static library (i.e. ``VCPKG_CRT_LINKAGE static``).
+
+The ``static-md`` linking triplets are set to use the MSVC Runtime as a DLL (i.e. ``VCPKG_CRT_LINKAGE dynamic``). This is the recommended solution for redistributing the MSVC Runtime per [Microsoft Learn](/cpp/windows/deployment-in-visual-cpp).
+
+## Selecting a Visual C++ Toolset
+
+By default, vcpkg will use the latest version of Visual Studio installed on the system for building code. To select a specific version, create a custom triplet or triplet overlay to set it.
+
+For examples, this would force the use of the VS 2017 toolset.
+
+```
+set(VCPKG_PLATFORM_TOOLSET v141)
+```
+
+See [Triplet files](../triplets.md) for more information.
+
+## C/C++ Runtime
+
+Keep in mind that the Microsoft Visual C/C++ Runtime is 'forward binary compatible'. This means you can build code with VS 2015 Update 3, VS 2017, VS 2019, and/or VS 2022 and link it all together. The key requirement is that the LINK must be done against the *newest* toolset in the mix. See [Microsoft Learn](/cpp/porting/binary-compat-2015-2017).
+
+## Known issues
+
+* If you get the following link failure, it is because you are building x64 native code with VS 2019 or later, but linking it with the VS 2017 toolset & CRT combination. Per the notes above, you should use the newest compiler toolset for your final link. This *specific* issue can be worked around as well with ``/d2FH4-`` (v142 or later) to use the older x64 C++ exception implementation.
+
+```
+error LNK2001: unresolved external symbol ___CxxFrameHandler4
+```
+
+See the [Visual C++ Team Blog](https://devblogs.microsoft.com/cppblog/making-cpp-exception-handling-smaller-x64/) for more information.
+
+* Full C++20 support requires VS 2019 (16.11.14 or later) and/or VS 2022 (17.2 or later). See the [Visual C++ Team Blog](https://devblogs.microsoft.com/cppblog/msvc-cpp20-and-the-std-cpp20-switch/).
+
+## Maintainer notes
+
+CMake projects for these triplets are built using ``CMAKE_SYSTEM_NAME`` set to "Windows".
+
+* "Just My Code" debugging can usually be disabled in a library to save code space.
```

Comment:
```suggestion
## Library author notes

* "Just My Code" debugging can usually be disabled in a library to save code space.
```

We typically reserve "maintainer" to refer to the owner of the port, which is different than the author of the library.